src/app/components/Toolbar/Features/AccountSelector/index.tsx

```diff
@@ -50,6 +51,10 @@ export const Account = memo((props: AccountProps) => {
     [WalletType.PrivateKey]: t('toolbar.wallets.type.privateKey', 'Private key'),
   }
 
+  const { details } = props
+
+  const index = details ? details.split('/').at(-1) : '?'
```

Comment:
I'm not a fan of the name `index` outside of loop iterators. accountNumber?

Reply:
OK, renamed.